packages/grpc-protobufjs/README.md

```diff
@@ -1,6 +1,8 @@
 # gRPC Protobuf Loader
 
 A utility package for loading `.proto` files for use with gRPC, using the latest Protobuf.js package.
+Please refer to [protobuf.js' documentation](https://github.com/dcodeIO/protobuf.js/blob/master/README.md)
+to understands its usage and limitations.
```

Comment:
I'm not sure about pointing to Protobuf.js for "usage". The usage of this library is defined in this documentation, not in the Protobuf.js documentation.

Reply:
I wanted to say something of the ilk of "pros and cons", but since we don't have any comparison point, I'm not sure this idiom would be appropriate. Any suggestion ?

Reply:
Its features, maybe?

Reply:
I think "features and limitations" would work here.